.github/workflows/website.yml

```diff
@@ -115,17 +115,15 @@ jobs:
       - uses: ./.github/actions/sccache
         with:
           key: Linux-default
-      - uses: risc0/foundry-toolchain@2fe7e70b520f62368a0e3c464f997df07ede420f
```

Comment:
is deleting this relevant to enabling lychee?

Reply:
This updates the workflow so it properly builds the 'stable' version of the docs. Something that should have happened when the docs got promoted but was neglected.